presence-detector.py

```diff
@@ -40,11 +40,16 @@ def __init__(self, config_file):
         self.clients_seen = {}
 
     def ha_seen(self, client, seen=True):
-        location = "home"
+        location = self.settings.zone
         if not seen:
-            location = "not_home"
+            location = "unknown"
```

Comment:
I'd rather make this f"not_{location}", so we don't break existing setups/automations

Reply:
How is this supposed to break existing automations? I think unknown is more accurate, since "not_{location}" defines a location, while it should not. You actually don't know where it is. I pose you this case.
I've got two floors, each of them with an access point, which runs the script. When I am on a floor, for the same MAC, a source reports "location1", while the other one reports "not_location2". Which one should I listen to?
Using "unknown" the first reports "location1" and the second "unknown". It should be only matter of choosing the only one known, instead of defining a logic among locations.

Reply:
Well, for instance my own automations trigger on a transition from "home" to "not_home", so they wouldn't trigger anymore with this change. But I see your point, making it not_{zone} doesn't make sense.

Reply:
What we could do is define another setting which states what you should say when you lose track of a device. Which can default either to "unknown" or "not_{location}".

Reply:
Yep, good idea

Reply:
Which one? :)

Reply:
Since zone defaults to home, I'd say default this to "not_home".

Reply:
And rename of "zone" to "location" makes sense, yes

Reply:
Let's go